cloudant.js

```diff
@@ -56,10 +56,11 @@ function Cloudant(options, callback) {
     theurl = reconfigure({ url: options });
   }
   if (theurl === null) {
+    var err = new Error('Invalid URL');
```

Comment:
Is it worth including the invalid/malformed URL value with this error message?

Reply:
I think it's okay. This all happens in the reconfigure.js code which uses `url.parse` to check for the necessary url bits (else returns null).